src/main/resources/page/js/dummy-endpoint-caller.js

```diff
@@ -0,0 +1,8 @@
+class APICaller {
+    constructor() {}
+
+    static async getDateAndTimeInJson(url) {
+        var dateAndTime = null;
+        return dateAndTime = await fetch(url).then(response => response.json());
```

Comment:
Is this expression valid?
`return await fetch(url).then(response => response.json()`);

Reply:
Yes, it is valid I will do it this way. :)